modules/survey_accounts/js/survey_accounts_helper.js

```diff
@@ -58,14 +57,35 @@ $(document).ready(function () {
     });
     $("input[type=submit]").click(function (e) {
         if(e.currentTarget.classList.contains('email')) {
-            $("#emailModal").modal();
+            $.get(loris.BaseURL + "/survey_accounts/ajax/ValidateEmailSubmitInput.php", {
+                dccid: $("input[name=CandID]").val(),
```

Comment:
@davidblader For future reference, to send the form data, all you have to do is something like that

```js
var formData = $('form[name=survey_accounts]').serialize();
$.get(loris.BaseURL + "/survey_accounts/ajax/ValidateEmailSubmitInput.php", data: formData);
```